website/docs/r/datadog_sso.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_sso"
+description: |-
+  Manages SingleSignOn on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_sso
```

Comment:
Can we rename this to `azurerm_datadog_monitor_sso_configuration`? Meanwhile, please also rename the filename for its resource and resource test files to has them prefixed by `datadog_monitor_sso_configuration`.

Reply:
Done